Final edits (#52)
greenelab/text_mined_hetnet_manuscript: content/04.results.md

### Generative Model Using Randomly Sampled Label Functions

Creating label functions is a labor-intensive process that can take days to accomplish.
We sought to accelerate this process by measuring the extent to which label functions can be reused.
Our hypothesis was that certain edge types share similar linguistic features such as keywords and/or sentence structure.
This shared characteristic would make certain edge types amenable to label function reuse.
We designed a set of experiments to test this hypothesis on an individual level (edge vs edge) as well as a global level (collective pool of sources). 

> We reported results in terms of AUROC (Figure {@fig:auroc_gen_model_test_set} and Supplemental Figure {@fig:auroc_grabbag_gen_model_test_set}) and AUPR (Supplemental Figures {@fig:aupr_gen_model_test_set} and {@fig:aupr_grabbag_gen_model_test_set}).

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

> Performance increases when edge-specific label functions are added to an edge-specific baseline model, while label function reusability shows modest results.

Reviewer: ```suggestion
We observed that performance increased when edge-specific label functions were added to an edge-specific baseline model, while label function reuse usually provided less benefit (AUROC Figure {@fig:auroc_gen_model_test_set}, AUPR Supplemental Figure {@fig:aupr_gen_model_test_set}).
We also evaluated randomly selecting label functions from among all sets and observed similar performance (AUROC Supplemental Figure {@fig:auroc_grabbag_gen_model_test_set}, AUPR Supplemental Figure {@ fig:aupr_grabbag_gen_model_test_set})
```
